samples/Gonda/Gonda.Server/Services/GreeterService.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Grpc.Core;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.Extensions.Logging;
+using OpenIddict.Validation.AspNetCore;
+
+namespace Gonda.Api1
```

Comment:
Now that the gRPC service and the authorization server are part of the same app, you may want to change all references to the `Gonda.Api1` namespace to something like `Gondo.Server.Services`.

Reply:
Good call